js/main.js

```diff
@@ -79,20 +73,22 @@ function handleDOMContentLoaded(event) {
   }
 }
 
-$viewSwapButton.addEventListener('click', handleClick);
+document.addEventListener('click', handleClick);
 function handleClick(event) {
-  for (var i = 0; i < $allViews.length; i++) {
-    if ($allViews[i].getAttribute('data-view') === 'entry-form') {
-      $allViews[i].className = 'view';
-
-    } else {
-      $allViews[i].className = 'view hidden';
+  if (event.target.matches('.view-switch') === true) {
+    for (var i = 0; i < $allViews.length; i++) {
+      if ($allViews[i].getAttribute('data-view') === 'entry-form') {
+        $allViews[i].className = 'view';
+      } else {
+        $allViews[i].className = 'view hidden';
+      }
```

Comment:
This looks like essentially the same exact code as `entrySwap`. Instead, I recommend making a function definition that can receive an argument of the name of the view to show (either `"entry-form"` or `"entries"`), and do this same loop over the views. If the current view in the loop matches the argument value, show that view. If not, hide that view.